tcpdevmem.c

```diff
@@ -11,6 +11,44 @@
 
 #define TEST_PREFIX "ncdevmem"
 
+int driver_reset(const struct options *opts) {
+	char driver_reset_cmd[512];
+	int ret = 0;
+
+	sprintf(driver_reset_cmd, "ethtool --set-priv-flags %s enable-strict-header-split on", opts->tcpd_link_name);
+	ret = ret | system(driver_reset_cmd);
+
+	sprintf(driver_reset_cmd, "ethtool --set-priv-flags %s enable-strict-header-split off", opts->tcpd_link_name);
+	ret = ret | system(driver_reset_cmd);
+
+	sprintf(driver_reset_cmd, "ethtool --set-priv-flags %s enable-header-split off", opts->tcpd_link_name);
+	ret = ret | system(driver_reset_cmd);
+
+	sprintf(driver_reset_cmd, "ethtool --set-rxfh-indir %s equal 16", opts->tcpd_link_name);
+	ret = ret | system(driver_reset_cmd);
+
+	sprintf(driver_reset_cmd, "ethtool -K %s ntuple off", opts->tcpd_link_name);
+	ret = ret | system(driver_reset_cmd);
+
+	sprintf(driver_reset_cmd, "ethtool --set-priv-flags %s enable-strict-header-split off", opts->tcpd_link_name);
+	ret = ret | system(driver_reset_cmd);
+
+	sprintf(driver_reset_cmd, "ethtool --set-priv-flags %s enable-header-split off", opts->tcpd_link_name);
+	ret = ret | system(driver_reset_cmd);
+
+	sprintf(driver_reset_cmd, "ethtool -K %s ntuple off", opts->tcpd_link_name);
+	ret = ret | system(driver_reset_cmd);
+
+	sprintf(driver_reset_cmd, "ethtool --set-priv-flags %s enable-max-rx-buffer-size on", opts->tcpd_link_name);
+	ret = ret | system(driver_reset_cmd);
+
+	sprintf(driver_reset_cmd, "ethtool -K %s ntuple on", opts->tcpd_link_name);
```

Comment:
I think you need to turn on header split back on somewhere in this function, right? Otherwise header split remains off and tcpx should not work, right?

Reply:
this happens in `tcpdevmem.c#install_flow_steering`